hello-world/example.rb

```diff
@@ -0,0 +1,5 @@
+class HelloWorld
+  def self.hello(name = '')
+    name == '' ? 'Hello, world!' : "Hello, #{name}!"
```

Comment:
This is a little bit "tricky". You already have a default value, why not just use it?

``` ruby
def self.hello(name = 'World')
  "Hello, #{name}!""
end
```

Least amount of complication.